arduino-ide-extension/src/browser/contributions/core-error-handler.ts

```diff
@@ -4,29 +4,29 @@ import { CoreError } from '../../common/protocol/core-service';
 
 @injectable()
 export class CoreErrorHandler {
-  private readonly compilerErrors: CoreError.Compiler[] = [];
+  private readonly errors: CoreError.ErrorLocation[] = [];
   private readonly compilerErrorsDidChangeEmitter = new Emitter<
-    CoreError.Compiler[]
+    CoreError.ErrorLocation[]
   >();
 
   tryHandle(error: unknown): void {
     if (CoreError.is(error)) {
-      this.compilerErrors.length = 0;
-      this.compilerErrors.push(...error.data.filter(CoreError.Compiler.is));
+      this.errors.length = 0;
+      this.errors.push(...error.data);
```

Comment:
Isn't this equivalent to
```suggestion
      this.errors = error.data;
```
performance wise is also faster

Reply:
The fields are `readonly`. Your proposal is a compiler error. Did I overlook something?